.infrastructure/11_dns.tf

```diff
@@ -1,4 +1,11 @@
 module "dns_zone" {
   source = "git::https://github.com/terraform-aws-modules/terraform-aws-route53.git//modules/zones?ref=bc63328714550fd903d2574b263833c9ce1c867e" # v2.11.0
   zones  = var.dns_domain_name
-}
\ No newline at end of file
+}
+
+resource "aws_route53_record" "cms" {
```

Comment:
can we use [the module](https://github.com/terraform-aws-modules/terraform-aws-route53/tree/master/modules/records) instead? what do you think?

Reply:
with last commit is used records and acm modules